docs/faq.md

```diff
@@ -19,6 +31,21 @@
     - [Setting up PMM Server](setting-up/server/index.md)
     - [Setting up PMM Client](setting-up/client/index.md)
 
+## How do I install it?
+
+1. [Set up a PMM Server](setting-up/server/index.md).
+1. On the host where your database is, [set up PMM Client](setting-up/client/index.md).
+1. [Configure the database according to its type](setting-up/client/index.md#add-services).
+1. [Register the client with the server](setting-up/client/index.md#register).
+1. [Open the PMM web UI to start seeing metrics appear](using/interface.md).
+
+## How can I contact the developers?
+
+- [Community forum](https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management).
+- [Discord chat](http://per.co.na/discord).
+- [PMM project in JIRA](https://jira.percona.com/projects/PMM).
+
+
 ## How can I upgrade from version 1?
 
 There is no direct software upgrade path.
```

Comment:
somewhere I found a link with more detail in the docs that talks about the architectural changes...

Here: can we link here for more info: how-to/upgrade.html#upgrade-from-pmm-1  (not sure how to make the anchor link with the .md extension...)